src/app/ubs/ubs-user/services/user-messages.service.ts

```diff
@@ -11,13 +11,13 @@ import { takeUntil } from 'rxjs/operators';
 })
 export class UserMessagesService implements OnDestroy {
   url = environment.backendUbsLink;
-  private destroyed$: ReplaySubject<any> = new ReplaySubject<any>(1);
-  countOfNoReadeMessages: any;
+  private readonly destroyed$: ReplaySubject<any> = new ReplaySubject<any>(1);
+  countOfNoReadMessages: any;
   language: string;
 
   constructor(
-    private http: HttpClient,
-    private localStorageService: LocalStorageService
+    private readonly http: HttpClient,
+    private readonly localStorageService: LocalStorageService
```

Comment:
it seems that the LocalStorageService is only being used to fetch the language parameter for the getNotification method, it makes sense to pass the language as a parameter in UbsUserMessagesComponent:
`getNotification(currentPage: number, size: number, language: string){};`
`this.fetchNotification(language);`